pkg/controllers/repos.go

```diff
@@ -21,6 +21,7 @@ func (h *handler) patchUpdateRegistration(incoming, target *v1.Registration) (*v
 		return incoming, err
 	}
 
+	// TODO: debug why this patch is causing issue snow
```

Comment:
I do notice this causing issues, but since this is called in a retrying function and eventually it works fine. So this TODO is one that isn't super urgent  (unless QA) finds issues as they test.